javascript.js

```diff
@@ -8,6 +8,7 @@ function mobileMenu() {
   document.getElementsByClassName('ham_menu')[2].style.display = 'flex';
 }
 mobileMenu();
+
 function desktopMob() {
   document.getElementsByClassName('name_head')[0].style.display = 'flex';
   document.getElementsByClassName('material-symbols-outlined')[0].style.display = 'flex';
```

Comment:
### Missing Requirements :m:
![missing-req (copy)](https://user-images.githubusercontent.com/93039359/179648718-b3207700-57bd-4e73-b5ea-d2ca6f42c41f.png)

- [ ] It's great that you have javascript and you know how you can manipulate your dom. :clap: but you have not implemented all the requirements. Those requirements have been shown in the image above. :arrow_up_small:

Reply:
Hello! no the menu dissapears